src/types.ts

```diff
@@ -17,22 +17,64 @@ export type SignedMessage = {
 };
 
 export type WallyConnectorOptions = {
-  // The clientId you get when signing up
+  /**
+   * The clientId you get when signing up. Should match the dashboard value
+   */
   clientId: string;
-  // Required when using devUrl or token
+  /**
+   * Required when using devUrl.
+   * TODO: Make this better.
+   * @default false
+   */
   isDevelopment?: boolean;
-  // The local instance of the wally api. APP ROOT used if unset or not dev mode
+  /**
+   * The local instance of the wally api. APP ROOT used if unset or not dev mode
+   */
   devUrl?: string;
-  // If you're testing stuff out in dev mode and don't have access to the
-  // login flow (i.e. chrome extension), but you do have a token, put it here
-  token?: string;
-  // If you want more insight into what's going on (requests and responses)
+  /**
+   * If you have another cross-tab messaging solution for informing the primary browser
+   * tab when the secondary
+   * @default false
+   */
+  disableSharedWorker?: boolean;
```

Comment:
Q: Just curious, in what scenario do we need this?

Reply:
Mostly just for the chrome extension - if there's a situation where the shared worker can't be used or isn't working, I thought it would be nice to just disable instead of spewing console errors

Reply:
I see and disabling won't break any intentional behaviors for the SDK?

Reply:
the SDK _works_ without it - the shared worker just informs the primary window when the secondary redirect has finished. the user can just refresh the main window and everything is fine

again, just for the chrome extension - i'll add a note

Reply:
Sounds good. This is non-blocking for me. Just want to understand it a little deeper. Don't know what has been done for the chrome extension overall.